CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ UNRELEASED
 * [ [#1322](https://github.com/digitalfabrik/integreat-cms/issues/1322) ] Add organization name & logo to the page API response
 * [ [#2105](https://github.com/digitalfabrik/integreat-cms/issues/2105) ] Indicate that categories are now shown in the apps
 * [ [#2065](https://github.com/digitalfabrik/integreat-cms/issues/2065) ] Only allow staff members to use SUMM.AI bulk action
+* [ [#1970](https://github.com/digitalfabrik/integreat-cms/issues/1970) ] Disable drag&drop in page order for users without permission
```

Comment:
```suggestion
* [ [#1970](https://github.com/digitalfabrik/integreat-cms/issues/1970) ] Disable drag & drop in page order for users without editing permission
* [ [#2112](https://github.com/digitalfabrik/integreat-cms/issues/2112) ] Hide form buttons and bulk actions for users without editing permissions
```
I think you can credit yourself for all the additional things you have fixed in this PR :grin: